requirements.txt

```diff
@@ -1,18 +1,26 @@
-colorama
-demjson3
+colorama~=0.4.6

```

Comment:
It added all the versions. I guess this needs to be modified at least into `>=` but I am not a Pythonian, I have no clue :)